src/Footer.scss

```
@@ -0,0 +1,14 @@
+.footer-main {
+    display: flex;
+    flex-direction: row; // make the child elements display horizontally
+    justify-content: space-between; // distribute the child elements to the left and right edges
+    align-items: center;
+    border: 1px solid blue;
+    width: 100%;
+    height: 70px;
+}
+.footer-left, .footer-right {
```

Blank line for spacing
```suggestion
}

.footer-left, .footer-right {
```